ultraLatency.py

```diff
@@ -182,6 +190,7 @@
         pass
     configureScanModule(ohboard, options.gtx, mode, mask,
                         scanmin=LATENCY_MIN, scanmax=LATENCY_MAX,
+                        stepsize=step,
```

Comment:
Sure, this is not nearly as important as this change being propagated to run_scans. We can not approve this PR until this happens.